core/web/helpers.go

```diff
@@ -28,6 +28,20 @@ func jsonAPIError(c *gin.Context, statusCode int, err error) {
 	}
 }
 
+func errorNotFound(c *gin.Context, err error) {
+	// once cleared, update to http.StatusNotFound
+	jsonAPIError(c, http.StatusInternalServerError, err)
+}
+
+func errorBadRequest(c *gin.Context, err error) {
+	// once cleared, update to http.StatusBadRequest
```

Comment:
Do we have a ticket tracking this work?
Normally we leave behind a comment like: `// TODO lorem ipsum https://jira.com/12345`

Reply:
Thanks! Will update